tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/closures/ClosureTest.java

```diff
@@ -273,4 +273,8 @@ public void testClosureWithinResource() {
         BRunUtil.invoke(result, "testClosureWithinResource");
     }
 
+    @Test(description = "Test error constructor within closure")
+    public void errorConstructorInClosureTest() {
+        BRunUtil.invoke(compileResult, "errorConstructorInClosureTest");
+    }
```

Comment:
```suggestion
    @Test(description = "Test error constructor with closure")
    public void errorConstructorWithClosureTest() {
        BRunUtil.invoke(compileResult, "errorConstructorWithClosureTest");
    }
```

because the error constructor itself captures variables.